### PR TITLE
get memory size

### PR DESCRIPTION
**Pull request by Z-bin**
get memory size and update README.md

### CODE REVIEW

File: src/qml/AboutPage.qml [hunk 1]
```diff
@@ -55,6 +55,11 @@ ItemPage {
                 value: about.kernelVersion
             }
 
+            StandardItem {
+                key: qsTr("Memory Size")
```
Comment (itsvic-dev) [inline src/qml/AboutPage.qml]: Could this be perhaps changed to "RAM Size"? I think it'd be more clear - some people still use "memory" to refer to storage medium, most commonly on mobile devices.

Comment (Z-bin) [reply src/qml/AboutPage.qml]: updated

**Review by itsvic-dev — CHANGES_REQUESTED** [review]
That's great, but it should be "RAM" instead of "Ram".

**Comment by Z-bin** [comment]
> That's great, but it should be "RAM" instead of "Ram".

updated again

**Pull request merged**